faker/providers/person/es_CA/__init__.py

```diff
@@ -0,0 +1,61 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from ..es_ES import Provider as PersonProvider
+
+
+class Provider(PersonProvider):
+    first_names_male = tuple(set(PersonProvider.first_names_male + (
```

Comment:
what's the source for the dataset? Please consult our [Coding style](https://github.com/joke2k/faker/blob/master/docs/coding_style.rst#data-sets) for guidelines about data sources

Reply:
@fcurella Yes, i’ve documented where i get the data from.